service_test.go

```diff
@@ -44,6 +47,59 @@ func TestService_List(t *testing.T) {
 	testEqual(t, want, res)
 }
 
+// ListServices
+func TestService_ListPaginated(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/services", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		offsetStr := r.URL.Query()["offset"][0]
+		offset, _ := strconv.ParseInt(offsetStr, 10, 32)
+
+		var more string
+		if offset == 0 {
+			more = "true"
+		} else {
+			more = "false"
+		}
+		resp := fmt.Sprintf(`{"services": [{"id": "%d"}],
+                          "More": %s,
+                          "Offset": %d,
+                          "Limit": 1}`, offset, more, offset)
+		w.Write([]byte(resp))
+	})
+
+	var listObj = APIListObject{Limit: 1, Offset: 0, More: false, Total: 0}
+	var client = &Client{apiEndpoint: server.URL, authToken: "foo", HTTPClient: defaultHTTPClient}
+	var opts = ListServiceOptions{
+		APIListObject: listObj,
+		TeamIDs:       []string{},
+		TimeZone:      "foo",
+		SortBy:        "bar",
+		Query:         "baz",
+		Includes:      []string{},
+	}
+	res, err := client.ListServicesPaginated(context.TODO(), opts)
```

Comment:
Sorry I missed this too, mind changing this to a `context.Background()`?

Reply:
You didn't miss this one - it's new code :-). I wasnt sure what context.Background does in a test, but sure.